algebra/archimedean.lean

```diff
@@ -220,6 +220,15 @@ let ⟨q, h₁, h₂⟩ := exists_rat_btwn $
   lt_trans ((sub_lt_self_iff x).2 ε0) ((lt_add_iff_pos_left x).2 ε0) in
 ⟨q, abs_sub_lt_iff.2 ⟨sub_lt.1 h₁, sub_lt_iff_lt_add.2 h₂⟩⟩
 
+lemma pow_unbounded_of_gt_one {x : α} (y : α) : 1 < x → ∃ n : ℕ, y < monoid.pow x n := begin
+  assume x1,have x0 : 0 < x - 1 := by rwa [lt_sub_iff,zero_add],
```

Comment:
usually hypothesis start with `h` (i.e. instead of `x1` call it `hx1`) Another option is to not give a name and refer to it using the french quotes (wirtten using `\f<` and `\f>`, so `‹1 < x›`)

Again `begin` on the next line.